packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -887,7 +890,10 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 		// get new DDS from first container
 		await provider.ensureSynchronized();
 		const dataStore1 = (await container1.getEntryPoint()) as ITestFluidObject;
-		const map2 = await requestFluidObject<SharedMap>(dataStore1.runtime, newMapId);
+		const containerRuntime = dataStore1.context.containerRuntime as ContainerRuntime;
+
+		const response = await containerRuntime.resolveHandle({ url: `/default/${newMapId}` });
```

Comment:
Same as above, this should not be using resolveHandle. You can do `dataStore1.runtime.getChannel(newMapId)` to get the map.

Reply:
Same for the rest of the usages in this file.